script.js

```diff
@@ -1,11 +1,57 @@
 
-getEverything({q: 'pole vault', source: 'CNN'}).then((response) => {
+getEverything({ q: 'pole vault', source: 'CNN' }).then((response) => {
   const { articles } = response;
+
+  // var articles = response.articles;
   const { url } = articles[0];
-  console.log({url, googleResults: response})
+
+
+  // var url = articles[0].url;
+  // console.log({ url, googleResponse: response })
+
+  renderTitles(articles)
+
   return getAnalysis({ url });
 }).then((response) => {
-  console.log({response})
+
+
 })
 
-// https://api.aylien.com/api/v1/combined?url=https://mashable.com/article/doomsday-vault-cherokee-nation/&endpoint=sentiment&endpoint=summarize
\ No newline at end of file
+// https://api.aylien.com/api/v1/combined?url=https://mashable.com/article/doomsday-vault-cherokee-nation/&endpoint=sentiment&endpoint=summarize
+
+
+function renderTitles(articles) {
+
+  //The purpose of this function is to get the title passed into the parameter and append it to the end of the headline-results list.
+  var articleList = $("#headline-results");
+
+
+  for (let i = 0; i < articles.length; i++) {
+
+    var articleTitle = articles[i].title;
+    var articleImage = articles[i].urlToImage;
+
+    //Create the HTML that create the card.
+    var cardtxt = "<div class='card horizontal orange darken-4 z-depth-5'> <div class='card-content white-text'>"
+      + "<div class='card-image'><img src=" + articleImage + "></div> " +
+
+      " <span class='card-title'>" + articleTitle + " </span></div></div>"
+
+    var cardtxt2 = "<div class='card horizontal orange darken-4 z-depth-5'> <div class='card-content white-text'>"
```

Comment:
Delete cardtxt2 please. This was there to make a comparison between having the article image on the left of the headline and having the headline over the image.